tasks/update-infra-deployments/update-infra-deployments.yaml

```diff
@@ -294,6 +294,14 @@ spec:
                 print(json_output)
 
         def main():
+
+            with open(r"updated_files.txt'", 'r') as ufiles:
+            updated_files = len(ufiles.readlines())
```

Comment:
I wonder how this works - this line should be indented relative to the `with` above.

Reply:
it doesn't :-1: 

new PR coming

Reply:
see https://github.com/redhat-appstudio/release-service-catalog/pull/260

Reply:
Great!